orca-webhook/src/test/groovy/com/netflix/spinnaker/orca/webhook/pipeline/PreconfiguredWebhookStageSpec.groovy

```diff
@@ -98,49 +98,12 @@ class PreconfiguredWebhookStageSpec extends Specification {
     ]
   }
 
-  def "Should prioritize user context to preconfigured context to allow overriding values in advanced use cases"() {
```

Comment:
The 1st test tests the new behavior already, so that's why i dont think we need this one.